### DISCUSSION TITLE
Howdy to Cisco-Images-for-GNS3-and-EVE-NG: Your Hub for Support, Resources, and Networking Excellence! 🌐🚀

### DISCUSSION THREAD

**Discussion by hegdepavankumar** [hegdepavankumar/Cisco-Images-for-GNS3-and-EVE-NG#3] [Q&A]
## Welcome to Cisco-Images-for-GNS3-and-EVE-NG 👋

Have questions? Need assistance? Looking for resources like vendor images 📸 or documents 📄? This is the place to be!

### How to Get Help:

Feel free to start  [Discussion](https://github.com/hegdepavankumar/Cisco-Images-for-GNS3-and-EVE-NG/discussions/3 ) for any questions or guidance you may need. Whether it's code-related, or you're on the hunt for useful documents, we've got you covered!

### What You Get from Us:

- **Comprehensive Vendor Images:** High-quality Cisco images for GNS3 and EVE-NG.
- **Extensive Documentation:** Well-documented guides and resources for seamless integration.
- **Responsive Support:** Open issues for any queries or assistance; we're here to help!

Your one-stop GitHub destination for Cisco images, support, and information! 🚀

**Comment by stevenjw0228001** [comment]
Any chance have the Cat9000v image?

**Comment by hegdepavankumar** [comment]
You can try this, updated one...
Link: [Click Here to Download](https://labhub.eu.org/api/raw/?path=/UNETLAB%20I/addons/qemu/cat9000v-17-10-01prd7/cat9kv-prd-17.10.01prd7.qcow2)

**Comment by nrgneilo** [comment]
Excellent find here.  Dell os10 image would be beneficial if possible?

**Reply by hegdepavankumar** [reply]
You can join our community: [Join Telegram](https://t.me/resourcehub1)

**Comment by hegdepavankumar** [comment]
1>  Dell OS 10 (dellos10-10.5.2.2.tgz) : [Click here](https://shorturl.at/epyzF)
2>  Dell OS 10 (dellos10-10.5.4.7.204.tgz) : [Click here](https://shorturl.at/lmoR2)
3> Dell OS 10 ([dellos10-10.5.5.5.105.tgz) : [Click Here](https://shorturl.at/cuxC4)

**Comment by hegdepavankumar** [comment]
> Excellent find here. Dell os10 image would be beneficial if possible?

**Comment by jvdholaria** [comment]
hi pavan,
can you pls share forescout counterACT img if possible...........

**Comment by hegdepavankumar** [comment]
ForeScout VM Requirements:
OS:  Linux
Ver:  Red Hat Enterprise Linux 6 (64-bit)
RAM: 16 GB
vCPU: 8 core
HDD: 300 GB (Thin Provision)
NIC: 2 interfaces 
ISO:  CounterACT-8.2.2.732
Image Download- 8.3 : [click here](https://mega.nz/file/DhQBDQzL#IqWpQX1FBs_Nj8lETY3icTPkJ2lcr-S7AzK1E5TwUkU)

**Reply by jvdholaria** [reply]
hi
can you also provide img for trendmicro and rsa authentication.....pls

**Comment by Nashovice1811** [comment]
Hi @hegdepavankumar 

can you provide me the correct cloud-init files for vmware velocloud orchestrator, gateway and edge devices (5.0.1,5.0.0 and 5.0.0)

i have tried the below configuration in orchestrator but i am able to login to cli mode but for GUI it shows invalid username and password 

meta-data

instance-id: vco01
local-hostname: vco-01
network-interfaces: |
  auto eth0
  iface eth0 inet dhcp


user-data

#cloud-config
            user: vcadmin
            password: Velocloud123
            chpasswd: {expire: False}
            ssh_pwauth: True
            vco:
              super_users:
                list: |
                  super@velocloud.com:Velocloud123
                remove_default_users: True
              system_properties:
                 list: |
                    mail.smtp.port:34
                    mail.smtp.host:[smtp.yourdomain.com](http://smtp.yourdomain.com/)
                    service.maxmind.enable:True
                    service.maxmind.license:todo_license
                    service.maxmind.userid:todo_user
                    service.twilio.phoneNumber:222123123
                    network.public.address:222123123

**Comment by TESLA1799** [comment]
Hi @hegdepavankumar 
I'm new to cybersecurity. I need some IP Phone images to see how the voice of one is being recorded through wireshark.
If possible please tell me about.
